ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -594,9 +586,11 @@ module.exports = Backbone.AssociatedModel.extend({
   },
 
   isInUpsSpace: function(lat, lon) {
+    lat = parseFloat(lat)
+    lon = parseFloat(lon)
     return (
-      (this.isLatLonValid(lat, lon) && (lat > 84 && lat < 90)) ||
-      (lat < -80 && lat > -90)
+      this.isLatLonValid(lat, lon) &&
+      ((lat > 84 && lat < 90) || (lat < -80 && lat > -90))
```

Comment:
checking for <90 and > -90 is redundant here as isLatLonValid should already cover that.
Consider using const rather than magic number 84 and -80